src/schema/v2/homeView/zones/__tests__/default.test.ts

```diff
@@ -23,6 +23,7 @@ describe("getSections", () => {
           "home-view-section-latest-activity",
           "home-view-section-new-works-for-you",
           "home-view-section-hero-units",
+          "home-view-section-explore-by",
```

Comment:
I am surprised about how high this section will be - auction lots for you and active bids are high converting rails that will massively get affected by this

Reply:
I believe all the related product teams + data will have to sit down and re-evaluate this order before releasing it 😅